pp.c

```diff
@@ -7157,6 +7158,42 @@ PP(pp_isa)
         RETURN;
     }
 
+    /* This abuse of gv_fetchmeth_pv() with level = 1 skips the UNIVERSAL
+     * lookup
+     * TODO: Consider if we want a NOUNIVERSAL flag for requesting this in a
+     * more obvious way
+     */
+    isagv = gv_fetchmeth_pv(SvSTASH(SvRV(left)), "isa", 1, 0);
```

Comment:
Can you use gv_fetchmeth_pvn instead of gv_fetchmeth_pv for a lesser run-time cost?

Reply:
It seems so. 04987d405a